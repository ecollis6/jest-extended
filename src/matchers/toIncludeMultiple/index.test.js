import matcher from './';

expect.extend(matcher);

test('passes when value includes all substrings', () => {
  expect('hello world').toIncludeMultiple(['world', 'hello']);
  expect('hello world').not.toIncludeMultiple(['world', 'hello', 'bob']);
});
