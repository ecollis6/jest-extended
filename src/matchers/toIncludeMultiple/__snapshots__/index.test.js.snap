// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`.toIncludeMultiple fails when string does not include all substrings 1`] = `
"<dim>expect(</><red>received</><dim>).toIncludeMultiple(</><green>expected</><dim>)</>

Expected string to contain all substrings: 
  <green>[\\"hello\\", \\"world\\", \\"bob\\"]</>
Received:
  <red>\\"hello world\\"</>"
`;

exports[`.toIncludeMultiple fails when string includes all substrings 1`] = `
"<dim>expect(</><red>received</><dim>).not.toInclueMultiple(</><green>expected</><dim>)</>

Expected string to not contain all substrings: 
  <green>[\\"hello\\", \\"world\\"]</>
Received: 
  <red>\\"hello world\\"</>"
`;
